app/javascript/src/Types/SwaggerTypes.jsx

```diff
@@ -0,0 +1,50 @@
+export type Primitive = string | number | boolean
+
+export type AccountData = {
+ [string]: Array<{name: string, value: string}> | []
```

Comment:
I think empty array is implicit:
```suggestion
 [string]: Array<{name: string, value: string}>
```